packages/blazor-workspace/Examples/Demo.Server/Properties/launchSettings.json

```diff
@@ -1,28 +1,38 @@
 {
```

Comment:
Is this a file that should actually be checked in? Doing a text search for all these ports I'm not sure where they are coming from, are they machine specific?

Reply:
I haven't seen any guidance online suggesting that it shouldn't be checked in. If you delete it entirely, launching the app at dev time (e.g. from Visual Studio, F5 or Ctrl-F5) don't work correctly, so I'm inclined to keep it as-is.

Not sure where the port numbers come from specifically. They do seem to vary across the various launchSettings.json files that I've made for the test Blazor+.NET8 projects I've created recently.